### PR TITLE
Completely fix MTOCRF

### PR DESCRIPTION
**Pull request by danilaml**
I hope. (Sound seems a little bit better too)

### CODE REVIEW

File: rpcs3/Emu/Cell/PPULLVMRecompilerCore.cpp [hunk 1]
```diff
@@ -2660,16 +2660,16 @@ void Compiler::MTOCRF(u32 l, u32 crm, u32 rs) {
 
 	for (u32 i = 0; i < 8; i++) {
 		if (crm & (1 << i)) {
-			mask |= 0xF << ((7 - i) * 4);
+			mask |= 0xF << (i * 4); // move 0xF to the left i positions (in hex form)
 			if (l) {
```
Comment (MrSapps) [inline rpcs3/Emu/Cell/PPULLVMRecompilerCore.cpp:2664]: if 1??

Comment (danilaml) [inline rpcs3/Emu/Cell/PPULLVMRecompilerCore.cpp:2664]: it's l.
`void Compiler::MTOCRF(u32 l, u32 crm, u32 rs)`

Comment (MrSapps) [inline rpcs3/Emu/Cell/PPULLVMRecompilerCore.cpp:2664]: How very confusing!

Comment (danilaml) [inline rpcs3/Emu/Cell/PPULLVMRecompilerCore.cpp:2664]: Yep. I couldn't find it in documentation too but it is there.
It's strange that GitHub's "programmers font" has that flaw of indistinguishable l and 1.

**Pull request merged**